websites/A/AWTWA/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "815513219120103424",
+		"name": "ember"
+	},
+	"service": "AWTWA",
+	"description": {
+		"en": "Discover the best free anime streaming site in high resolution.",
+		"fr": "Découvrez le meilleur site de streaming d'anime gratuit en haute résolution."
+	},
+	"url": [
+		"awtwa.site",
+		"www.awtwa.site"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/AoJisOX.png",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).